README.md

```diff
@@ -1,77 +1,28 @@
-
-Also, don't generate random rolls. Trust us on this one.
-
-### Optional Extras
-
-In any order you like:
-
-* Create a nice interactive animated interface with jQuery.
-* Set up [Travis CI](https://travis-ci.org) to run your tests.
-* Add [ESLint](http://eslint.org/) to your codebase and make your code conform.
-
-You might even want to start with ESLint early on in your work — to help you
-learn Javascript conventions as you go along.
-
-## Bowling — how does it work?
-
-### Strikes
-
-The player has a strike if he knocks down all 10 pins with the first roll in a frame. The frame ends immediately (since there are no pins left for a second roll). The bonus for that frame is the number of pins knocked down by the next two rolls. That would be the next frame, unless the player rolls another strike.
-
-### Spares
-
-The player has a spare if the knocks down all 10 pins with the two rolls of a frame. The bonus for that frame is the number of pins knocked down by the next roll (first roll of next frame).
-
-### 10th frame
-
-If the player rolls a strike or spare in the 10th frame they can roll the additional balls for the bonus. But they can never roll more than 3 balls in the 10th frame. The additional rolls only count for the bonus not for the regular frame count.
-
-    10, 10, 10 in the 10th frame gives 30 points (10 points for the regular first strike and 20 points for the bonus).
-    1, 9, 10 in the 10th frame gives 20 points (10 points for the regular spare and 10 points for the bonus).
-
-### Gutter Game
-
-A Gutter Game is when the player never hits a pin (20 zero scores).
+## My application
 
-### Perfect Game
+Currently, you can choose your score for each bowl, and the total for that frame will be calculated at the end of that frame (not including strike and spare scores.)
 
-A Perfect Game is when the player rolls 12 strikes (10 regular strikes and 2 strikes for the bonus in the 10th frame). The Perfect Game scores 300 points.
+I ran out of time to calculate a total and integrate spare and strike logic.
 
-In the image below you can find some score examples.
+## Personal Reflection
 
-More about ten pin bowling here: http://en.wikipedia.org/wiki/Ten-pin_bowling
+This was a tough week for me. I started by thinking about what the application would look like, and had a complete mental block on how a person would interact with it. I was unable for some reason to see past just randomising the output of rolls, and only realised late Sunday evening of the first week that my interface could be interactive.
 
-![Ten Pin Score Example](images/example_ten_pin_scoring.png)
+I struggle greatly with planning, and feel lost until I start coding, and I am not really sure how to approach full stack applications like this in terms of planning. Should I have planned out the interface first and worked back to the logic? I started on the logic first as this was what was most comfortable for me as I was not used to browser-based interaction.
```

Comment:
I would probably take the same approach.  In this case, I would expect your front end interactions to be pretty dependent on your logic, so this seems like the natural order to do things in.  For example, if you have a button which, when clicked, represents the number of pins knocked down, its not going to be able to do anything unless you can hook it up to a function which deals with knocking down pins.